src/NzbDrone.Core/Indexers/EzrssTorrentRssParser.cs

```diff
@@ -20,6 +18,11 @@ public EzrssTorrentRssParser()
 
         protected override bool PreProcess(IndexerResponse indexerResponse)
         {
+            if (indexerResponse.HttpResponse.HasHttpError)
```

Comment:
I think we always want to call this, same as we did before to catch HTML content responses.

Reply:
I understand, but EZRSS parser always checked this, unless there is a report of an issue then it should continue to do so.

Reply:
You don't want to revert it, just don't require an HTTP error to check it here.

Reply:
Only for EZRSS, it was the only one previously using `base.PreProcess`